_config.yml

```diff
@@ -35,3 +35,9 @@ omefiles:
 
 omecmakesuperbuild:
  version: 0.5.0
+
+omeroinsight:
```

Comment:
An alternative YML construction here would be

```
omero:
  version: 5.4.10
  build: b105
  insight:
    version: 5.5.0-m3
  matlab
    version: 5.5.0-m1
```

i.e. it would express the collection of OMERO components and their latest versions in the 5.5 series

Reply:
Does it make sense to unify this with https://github.com/ome/bio-formats-build/blob/master/scripts/repositories.yml ? i.e.

```
name: omero
version: 5.4.10
base-branch: master
submodules:
  insight:
    version: ...
    base-branch: master
```